nix/mobile/android/release.nix

```diff
@@ -79,6 +79,8 @@ in stdenv.mkDerivation rec {
   STATUS_GO_SRC_OVERRIDE = statusGoSrcOverride;
   ANDROID_ABI_SPLIT = androidAbiSplit;
   ANDROID_ABI_INCLUDE = androidAbiInclude;
+  # hermes is Disabled by default, we enable it for release https://github.com/status-im/status-mobile/issues/18493
+  ORG_GRADLE_PROJECT_hermesEnabled = true;
```

Comment:
```suggestion
  # Disabled for debug builds to avoid 'maxiumum call stack exceeded' errors.
  # https://github.com/status-im/status-mobile/issues/18493
  ORG_GRADLE_PROJECT_hermesEnabled = true;
```